source/layouts/fuse/_how_it_works.html.haml

```diff
@@ -0,0 +1,10 @@
+%section#howitworks.bg-navy.pv4
```

Comment:
My bad, wrong section. Thought this was nonprofits but see you haven't done that yet.